Main/ClientApp/src/Authorization/AuthenticatedView.tsx

```diff
@@ -50,6 +63,17 @@ export default function AuthenticatedView(props: AuthenticatedViewProps) {
         <IconButton color="inherit" onClick={openMenu}>
           <AccountCircle />
         </IconButton>
+        {/* https://ecstatic-elion-0c620b.netlify.app/dark-theme/ */}
+        <Switch
```

Comment:
Why not just use material.ui's switch?

Reply:
because i wanted a fancy one
when you run it you will see why XD